### ISSUE TITLE
TypeError: $scope.isValidMoment is not a function

### ISSUE THREAD

**Issue by tayutaedomo** [indrimuska/angular-moment-picker#99]
I found a error on following code.
https://github.com/indrimuska/angular-moment-picker/blob/master/src/angular-moment-picker.js#L761

I tried to fix this error. How about is below code?
```
                                $scope.$watch('locale', function (locale, previous) {
                                        if (!angular.isDefined(previous) || locale == previous) return;
-                                       if ($scope.isValidMoment($ctrl.$modelValue)) $scope.utility.setValue($ctrl.$modelValue.locale(locale));
-                                       if ($scope.isValidMoment($scope.limits.minDate)) $scope.limits.minDate = $scope.limits.minDate.locale(locale);
-                                       if ($scope.isValidMoment($scope.limits.maxDate)) $scope.limits.maxDate = $scope.limits.maxDate.locale(locale);
+                                       if ($scope.utility.isValidMoment($ctrl.$modelValue)) $scope.utility.setValue($ctrl.$modelValue.locale(locale));
+                                       if ($scope.utility.isValidMoment($scope.limits.minDate)) $scope.limits.minDate = $scope.limits.minDate.locale(locale);
+                                       if ($scope.utility.isValidMoment($scope.limits.maxDate)) $scope.limits.maxDate = $scope.limits.maxDate.locale(locale);
                                        $scope.view.render();
                                });
```

**Comment by indrimuska** [comment]
Thank you @tayutaedomo for reporting this, this is a regression introduced by last refactoring.
Give me 5 to make the changes.

**Comment by indrimuska** [comment]
Fixed in [v0.8.3](https://github.com/indrimuska/angular-moment-picker/releases/tag/0.8.3).
